solo-chains/runtime/starlight/src/tests/common/mod.rs

```diff
@@ -240,9 +240,8 @@ pub fn end_block() {
 }
 
 pub fn run_block() {
-    end_block();
-
-    start_block()
+    start_block();
+    end_block()
```

Comment:
I am not sure if we want this change, what was the reasoning behind it?

Reply:
We want to mimic the closest block production routine possible. WIthout this change we have situations where we have tests that require to manually end blocks because there are changes that happen on_finalize. So yes I think this change is worth pursuing